File: UNDPServer/CarRequest/Entities/TripDTO.cs
using Shared.Enums;
using System;
using System.Collections.Generic;
using System.Text;

namespace FleetManagement.Entities
{
    public class TripDTO
    {
        public long Id { get; set; }
        public TripStatusEnum TripStatusId { get; set; }
        public int PreviousKiloMeterCounter { get; set; }
        public int CurrentKiloMeterCounter { get; set; }
        public DateTime ActualStartTime { get; set; }
        public DateTime ActualEndTime { get; set; }
        public TripClosingReasonEnum TripClosingReasonId { get; set; }
        public long CarRequestId { get; set; }
        public long DriverId { get; set; }
        public string DriverName { get; set; }
        public string SequenceNumber { get; set; }
        public long CarId { get; set; }
        public string CarBrand { get; set; }
        public string CarNumber { get; set; }

    }
}

File: UNDP-app/src/app/modules/fleet-management/models/trip-dto.ts
import { TripClosingReasonEnum } from "src/app/shared/enums/trip-closing-reason-enum";
import { TripStatusEnum } from "src/app/shared/enums/trip-status-enum";

export class TripDTO{
      id : number;
      tripStatusId : TripStatusEnum
      previousKiloMeterCounter : number;
      currentKiloMeterCounter : number;
      actualStartTime : string
      actualEndTime : string
      tripClosingReasonId : TripClosingReasonEnum
      sequenceNumber :  number;
      driverId :  number;
      driverName : string
}